app/controllers/dashboard/tasks_controller.rb

```diff
@@ -0,0 +1,173 @@
+module Dashboard
+  class TasksController < ApplicationController
```

Comment:
**Critical - Code Quality**
This controller is fat. Let's strive to have at most 100 lines per class.

Please let's slim down this class.